test/support/serialization_testing.rb

```diff
@@ -10,4 +17,13 @@ def with_adapter(adapter)
   ensure
     ActiveModel::Serializer.config.adapter = old_adapter
   end
+  alias_method :with_configured_adapter, :with_adapter
```

Comment:
Oh, I got it. I was reading that as "do stuff with the adapter I'm providing, that I have given a specific configuration", instead of "do stuff with this adapter by setting it as the configured adapter".